prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -0,0 +1,217 @@
+---
+name: waters-covid-19
+description: WATERS OTC,POC COVID-19 flat file
+topic: covid-19-otc
+trackingElement: equipment_instance_id
+basedOn: covid-19
+elements:
+#  I removed this - we can pull it from the equipment_model_name lookup
+#  - name: ordered_test_code
+#    csvFields: [{ name: testOrdered}]
+
+  # Edited. Added 'ordering_facility_state', because it lets us test mapping to many of our states
+  - name: ordering_facility_state
+    cardinality: ZERO_OR_ONE
+    csvFields: [{ name: orderingFacilityState }]
+
+  - name: test_result    # Edited.  Was 'patient_positive' to 'test_result'
+    altValues:
+      - display: Patient Tested Positive!
+        code: 260373001
+      - display: Patient Tested Negative!
+        code: 260415000
+      - display: Test Was Invalid
+        code: 455371000124106
+    csvFields: [{ name: testResult, format: $alt}]
+
+  - name: test_result_date  # Edited.  Was 'patient_results' to 'test_result_date'
+    csvFields: [{ name: testResultDate}]
+
+  - name: test_result_report_date
+    csvFields: [{ name: testReportDate}]
+
+  - name: specimen_collection_date_time
+    csvFields: [{ name: specimenCollectedDate}]
+
+  # Edited.  This appeared twice.  I eliminated one.  The other was csvField = 'deviceName'
+  #  Note:  value here should come from LIVD table
+  - name: equipment_model_name
+    csvFields: [{ name: deviceIdentifier, format: $display}]
+
```

Comment:
This should be `device_id` right? In LVID, the device_id is different from equipment_model_name, so turning it into an equipment_model_name will not work. 

The big problem that I foresee is that device_id may be the natural index for the LVID table when we receive HL7 messages (it is the OBX-17 value). However, it is not in the LVID table. Let's talk about how to proceed.

Reply:
Both Strac and SR take in equipment_model_name, and so it seems like the easiest thing is to ask that Waters Senders do the same.   So I've changed the csvfieldname to equipmentModelName.  

...and fwiw, the equipment_model_name column for SR is labelled 'Device_ID' :-(